xsl/xccdf-report-impl.xsl

```diff
@@ -491,6 +491,18 @@ Authors:
         <xsl:when test="starts-with($href, 'https://www.fbi.gov/file-repository/cjis-security-policy')">
             <xsl:text>FBI CJIS</xsl:text>
         </xsl:when>
+        <xsl:when test="starts-with($href, 'http://www.ssi.gouv.fr/administration/bonnes-pratiques')">
+            <xsl:text>ANSSI</xsl:text>
+        </xsl:when>
+        <xsl:when test="starts-with($href, 'https://www.gpo.gov/fdsys/pkg/CFR-2007-title45-vol1')">
+            <xsl:text>HIPAA</xsl:text>
+        </xsl:when>
+        <xsl:when test="starts-with($href, 'https://www.iso.org/standard')">
```

Comment:
let's be more conservative when checking for URLs, this will pick any ISO standard

Reply:
@mpreisler done